packages/alfa-rules/src/sia-r66/rule.ts

```diff
@@ -54,7 +84,10 @@ export default Rule.Atomic.of<Page, Text, Question.Metadata>({
             return;
           }
 
-          if (test(and(isText, isPerceivableForAll(device)), node)) {
+          if (
+            test(and(isText, isPerceivableForAll(device)), node) &&
+            !disabledWidgetTexts.has(node)
```

Comment:
It doesn't quite work with the `isText` as the `node` is still type of `Node` there and the compiler is not happy because in `disabledWidgetTexts` there are `Text`s

Reply:
Ah yes, the first `and(isText, …)` should be a `Refinement.and` so that the second `and(isPerceivable, not(set.has))` only receives `Text`. 🤔 (also maybe wrong parenthesis in the suggestion 🙃 should be `and(isText, and(isPerceivable, not(set.has)))`)